### ISSUE TITLE
HTTP cache control headers (iOS RDServices, needs back-porting to OSX)

### ISSUE THREAD

**Issue by danielweck** [readium/SDKLauncher-OSX#47]
See:
https://github.com/readium/readium-sdk/pull/236

**Comment by olivierkorner** [comment]
@danielweck Just to be sure: this launcher doesn't use the RDServices of the readium-sdk submodule?
I have to copy the files in this repo?

**Comment by danielweck** [comment]
@olivierkorner correct, RDServices is only used by the iOS launcher app, unfortunately. I started "porting" the OSX app...but it's a pretty big job! See https://github.com/readium/SDKLauncher-OSX/tree/feature/RDServices
